v63003/outref/gtm8844.txt

```diff
@@ -0,0 +1,85 @@
+# Running functions that violate our restrictions
+# -------------------------------------------------------------------------------------------
+# Two consecutive halts
+# Halt 1, Expect a RESTRICTEDOP error
+ZSTATUS=haltfn+3^gtm8844,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: HALT
+# Halt 2
+%YDB-F-RESTRICTEDOP, Attempt to perform a restricted operation: HALT
+##TEST_AWKYDB_FATAL_ERROR.ZSHOW_DMP_[0-9]*_1.txt
+# -------------------------------------------------------------------------------------------
+# Two consecutive zhalts
+# Zhalt 1, Expect a RESTRICTEDOP error
+ZSTATUS=zhaltfn+3^gtm8844,%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: ZHALT
+# Zhalt 2
+%YDB-F-RESTRICTEDOP, Attempt to perform a restricted operation: ZHALT
+##TEST_AWKYDB_FATAL_ERROR.ZSHOW_DMP_[0-9]*_1.txt
```

Comment:
Can you add a comment line that we expect to see a fatal zshow dump file here. Likewise for the other cases where we do and don't expect it.